blocks/identity-block/features/account-management/_children/social-editable-section-styles.scss

```diff
@@ -0,0 +1,19 @@
+.social-editable-section--container {
+  display: flex;
+  align-items: center;
+
+}
+
+.social-editable-section--left-item {
+  flex: 1;
+  display: flex;
+  justify-content: flex-end;
+  padding: calculateRem(4px);
+}
+
+.social-editable-section--right-item {
+  flex: 1;
+  display: flex;
+  align-items: flex-start;
+  padding: calculateRem(4px);
+}
```

Comment:
With the removal of the additional CSS from `left` and `right` it's possible to consolidate the CSS into a single class for both

Reply:
really? we want this? can remove the align-items flex-start for the right item 

<img width="723" alt="Screen Shot 2021-12-10 at 09 14 49" src="https://user-images.githubusercontent.com/5950956/145596912-30cf6dac-39f5-4bc6-8cc1-4da92019b14b.png">

Reply:
```scss
.social-editable-section--container {
  display: flex;
  align-items: center;

}

.social-editable-section--left-item {
  flex: 1;
  // display: flex;
  // justify-content: flex-end;
  padding: calculateRem(4px);
}

.social-editable-section--right-item {
  flex: 1;
  // display: flex;
  // align-items: flex-start;
  padding: calculateRem(4px);
}
```